src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -290,7 +290,7 @@ struct Serializer
                         // Recurse into static array elements which contain a
                         // dynamic array.
 
-                        foreach (element; s.tupleof[i])
```

Comment:
This change (and other similar ones, below) looks unrelated to "Use `ref` iteration over struct fields".